build.sbt

```diff
@@ -323,6 +323,7 @@ lazy val bloopgun: Project = project
       Dependencies.coursierCache,
       Dependencies.jsoniterCore,
       Dependencies.jsoniterMacros % Provided,
+      "org.scalameta" %% "svm-subs" % "20.1.0" % "compile-internal", // Just giving this a try....
```

Comment:
Seems to work in Windows only (!). Is the subs versions coupled with graal version? Maybe it should be 20.2.0?

Reply:
:/ nope, same result.

Reply:
ah ha! teaches me to just see the three failures and _assume_ it's the same. 🤔

Reply:
I would try to print the classes that are problematic and then maybe add it to :
` shadeIgnoredNamespaces := Set("com.google.gson", "org.slf4j"),`

?